packages/harmony/src/components/button/SocialButton/SocialButton.module.css

```diff
@@ -1,34 +1,23 @@
 .root {
   border: transparent;
-  & svg path {
-    fill: var(--harmony-static-white);
-  }
+}
+
+.root svg path {
+  fill: var(--harmony-static-white);
 }
 
 .instagram {
   /* Due to the gradient we have to manually apply the same darken calculation that the button uses to each color in the gradient. 
        A bit hacky but this button is a unique edge case*/
-  background: radial-gradient(
-      61.46% 59.09% at 36.25% 96.55%,
-      color-mix(in srgb, var(--overlay-color) var(--overlay-opacity), #ffd600)
-        0%,
-      color-mix(in srgb, var(--overlay-color) var(--overlay-opacity), #ff6930)
-        48.44%,
-      color-mix(in srgb, var(--overlay-color) var(--overlay-opacity), #fe3b36)
-        73.44%,
-      rgba(254, 59, 54, 0) 100%
-    ),
-    radial-gradient(
-      202.83% 136.37% at 84.5% 113.5%,
-      color-mix(in srgb, var(--overlay-color) var(--overlay-opacity), #ff1b90)
-        24.39%,
-      color-mix(in srgb, var(--overlay-color) var(--overlay-opacity), #f80261)
-        43.67%,
-      color-mix(in srgb, var(--overlay-color) var(--overlay-opacity), #ed00c0)
-        68.85%,
-      color-mix(in srgb, var(--overlay-color) var(--overlay-opacity), #c500e9)
-        77.68%,
-      color-mix(in srgb, var(--overlay-color) var(--overlay-opacity), #7017ff)
-        89.32%
-    );
+  background: radial-gradient(61.46% 59.09% at 36.25% 96.55%,
+      color-mix(in srgb, var(--overlay-color) var(--overlay-opacity), #ffd600) 0%,
+      color-mix(in srgb, var(--overlay-color) var(--overlay-opacity), #ff6930) 48.44%,
+      color-mix(in srgb, var(--overlay-color) var(--overlay-opacity), #fe3b36) 73.44%,
+      rgba(254, 59, 54, 0) 100%),
+    radial-gradient(202.83% 136.37% at 84.5% 113.5%,
+      color-mix(in srgb, var(--overlay-color) var(--overlay-opacity), #ff1b90) 24.39%,
+      color-mix(in srgb, var(--overlay-color) var(--overlay-opacity), #f80261) 43.67%,
+      color-mix(in srgb, var(--overlay-color) var(--overlay-opacity), #ed00c0) 68.85%,
+      color-mix(in srgb, var(--overlay-color) var(--overlay-opacity), #c500e9) 77.68%,
+      color-mix(in srgb, var(--overlay-color) var(--overlay-opacity), #7017ff) 89.32%);
```

Comment:
We'll have to address this for the same reason, but it's not being used anywhere yet so will fix later